### PR TITLE
Satyam Commit from b1

### PR DESCRIPTION
**Pull request by SatyamSonje**
I have made some changes please review them

### CODE REVIEW

File: file1.js [hunk 1]
```diff
@@ -7,5 +7,8 @@ function sub(x,y) {
 function mult(x,y) {
     return ( x*y );
 }
+function div(x,y) {
```
Comment (KnoxLany) [inline file1.js:10]: you didnt return div in return default please correct it

Comment (SatyamSonje) [reply file1.js:10]: Nhi krna Mereko..Kya kr loge???

Comment (KnoxLany) [reply file1.js:10]: You are fired...

Comment (SatyamSonje) [reply file1.js:10]: ja be tu muze hi kaam nhi krna yaha pe....

**Pull request merged**